modules/oe_paragraphs_document/config/install/field.field.paragraph.oe_document.field_oe_media.yml

```diff
@@ -0,0 +1,32 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_oe_media
+    - media.type.document
+    - paragraphs.paragraphs_type.oe_document
+  module:
+    - composite_reference
+third_party_settings:
+  composite_reference:
+    composite: false
+id: paragraph.oe_document.field_oe_media
+field_name: field_oe_media
+entity_type: paragraph
+bundle: oe_document
+label: Document
+description: 'Provides Document paragraph that renders media documents.'
+required: true
+translatable: true
```

Comment:
This must be set to false, websites will flag it as translatable based on needs.